contrib/aws/gen-json.py

```diff
@@ -101,9 +101,6 @@
 # are started
 data['coreos']['units'] = new_units + data['coreos']['units']
 
-# configure etcd to use its EBS volume
-data['coreos']['etcd']['data-dir'] = '/media/etcd'
```

Comment:
@carmstrong yes, here https://github.com/deis/deis/pull/4024/files#diff-7b5fafac1ab3baf928f663c2b0a1a445R93

@mboersma I think that is one of the commits not selected